compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ClosureDesugar.java

```diff
@@ -972,7 +972,6 @@ public void visit(BLangXMLElementLiteral xmlElementLiteral) {
 
     @Override
     public void visit(BLangXMLTextLiteral xmlTextLiteral) {
-        xmlTextLiteral.textFragments.forEach(this::rewriteExpr);
```

Comment:
Is the branch in sync with master? This change was introduce in https://github.com/ballerina-platform/ballerina-lang/pull/33821/files#diff-7c71f66d9ff2b8ca170333cd7ee5f3456d84e3d4341a112f97aa323d3037315cL975.